<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineLearningPrediction</class>
 <widget class="QWidget" name="MachineLearningPrediction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MachineLearningPrediction</string>
  </property>
  <widget class="QFrame" name="mlp_frame">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>461</width>
     <height>531</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prediction</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>391</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plot_prediction"/>
   </widget>
  </widget>
  <widget class="QFrame" name="files_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_load">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_file">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start_prediction">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Prediction</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="random_forest_frame">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>60</y>
     <width>481</width>
     <height>531</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Confusion Matrix</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>411</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plot_confusion_matrix"/>
   </widget>
  </widget>
  <widget class="QTableWidget" name="table_widget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>401</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>401</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>401</width>
     <height>16777215</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string>Label</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prediction Table</string>
   </property>
  </widget>
  <widget class="QFrame" name="files_frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_load_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_model">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_model_name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Prediction Files</string>
   </property>
  </widget>
  <widget class="QFrame" name="files_frame_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>630</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_load_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_result_file">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Result Files and Plot</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_result_file">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>610</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Prediction Files</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_data_open">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Processing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ml_prediction_open">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Machine Learning Prediction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ml_open">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Machine Learning</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
